runtime/util/fltconv.c

```diff
@@ -277,8 +277,10 @@ jfloat
 helperCConvertIntegerToFloat(I_32 src)
 {
 	jfloat tmpDst;
-	
-#if defined USE_NATIVE_CAST
+
+#if defined(USE_NATIVE_CAST) || defined(ARM)
+	// Enabling native cast for ARM avoids a compiler bug in
```

Comment:
We avoid `//` comments in C code due to issues with some compilers.  The C code should still be compliant with `c89` due to certain supported platforms.

Reply:
ok, will do.